sigs.yaml

```diff
@@ -648,6 +648,20 @@ sigs:
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://docs.google.com/document/d/16ils69KImmE94RlmzjWDrkmFZysgB2J4lGnYMRN89WM/edit#
       recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
+    - description: Cluster API Implementer Monday Office Hours
```

Comment:
Do we want to keep this if no one is showing up.  TBH It's kind of ridiculous how many cluster-api meetings there are right now.  

/assign @timothysc 
/cc @roberthbailey @errordeveloper